### PR TITLE
Feature: added home  page

### PR DESCRIPTION
**Pull request by poojakelkar**
- Added E-Commerce Website Home page

Live-Link: **https://neog-e-comm-website.netlify.app/home/index.html**

### CODE REVIEW

File: home/homestyles.css [hunk 1]
```diff
@@ -1,97 +1,109 @@
...
+    border: 3px solid var(--grey-color-font);
+}
+.image-footer{
+    position: absolute;
+    top: 50.5rem;
+    width: 20rem;
+    border-radius: 10px;
+    text-align: center;
+    background-color: var(--light-white);
+    border: 3px solid gray;
```
Comment (arjunbharti) [inline home/homestyles.css:79]: use color variable here

**Review by arjunbharti — COMMENTED** [review]
Just a minor change required, everything else looks good to me

File: home/homestyles.css (hunk 1 above)
Comment (poojakelkar) [reply home/homestyles.css:79]: Thanks, @arjunbharti  for a suggestion.

**Review by sharmayatendra — APPROVED** [review]
just a small suggestion you can use the "alt" attribute for the images you have used. else it's good to go.

**Pull request merged**